protos/perfetto/perfetto_sql/structured_query.proto

```diff
@@ -374,4 +403,36 @@ message PerfettoSqlStructuredQuery {
     optional string column_name = 1 [deprecated = true];
   }
   repeated SelectColumn select_columns = 10;
+
+  // Represents an ORDER BY operation in SQL. Optional.
+  message OrderBy {
```

Comment:
At this point we need to define an evaluation order to these operators. Probably would suggest having a terse comment and then pointing to SQL standard for detaisl.